src/test/java/teammates/test/cases/ui/browsertests/FeedbackRankQuestionUiTest.java

```diff
@@ -423,11 +423,13 @@ public void testEditQuestionAction() throws Exception {
     public void testDeleteQuestionAction() {
         ______TS("rank: qn delete");
```

Comment:
Seems strange that there's no test for clicking cancel on the modal, but changing this is probably outside the scope of this PR.